backend/api/projects/resources.py

```diff
@@ -520,6 +520,9 @@ def setup_search_dto(self):
             search_dto.mapping_types = map(
                 str, mapping_types_str.split(",")
             )  # Extract list from string
+        search_dto.mapping_types_exact = strtobool(
```

Comment:
On the line 572, you need to add the new field on the view's docstrings, so it will generate the documentation on Swagger